bricks/template/__brick__/{{project_name.snakeCase()}}/test/mocks/generate_mocks.dart

```diff
@@ -1,10 +1,12 @@
 import 'package:dio/dio.dart';
 import 'package:{{project_name.snakeCase()}}/api/api_service.dart';
+import 'package:{{project_name.snakeCase()}}/api/repository/credential_repository.dart';
 import 'package:mockito/annotations.dart';
 
 @GenerateMocks([
   ApiService,
   DioError,
+  CredentialRepository,
```

Comment:
Could you please move it up?

```suggestion
  ApiService,
  CredentialRepository,
  DioError,
```